gtsfm/runner/run_scene_optimizer_astronet.py

```diff
@@ -86,6 +86,7 @@ def run_scene_optimizer() -> None:
         camera_intrinsics_graph=loader.create_computation_graph_for_intrinsics(),
         image_shape_graph=loader.create_computation_graph_for_image_shapes(),
         gt_pose_graph=loader.create_computation_graph_for_poses(),
```

Comment:
same comment everywhere: do we need both gt_poses and gt_cameras?

Reply:
removed it